webserver/src/main/java/io/kestra/webserver/controllers/FlowController.java

```diff
@@ -129,7 +130,7 @@ public PagedResults<SearchResult<Flow>> source(
     }
 
     @ExecuteOn(TaskExecutors.IO)
-    @Post(produces = MediaType.TEXT_JSON)
+    @Post(consumes = MediaType.ALL, produces = MediaType.TEXT_JSON)
```

Comment:
```suggestion
    @Post(consumes = MediaType.TEXT_PLAIN, produces = MediaType.TEXT_JSON)
```

Or maybe there is a YAML mime type ?